nodelet_topic_tools/include/nodelet_topic_tools/nodelet_lazy.h

```diff
@@ -217,22 +232,29 @@ class NodeletLazy: public nodelet::Nodelet
     */
   template<class T> ros::Publisher
   advertise(ros::NodeHandle& nh,
-            std::string topic, int queue_size, bool latch=false)
+            const std::string& topic, int queue_size, bool latch=false)
   {
     boost::mutex::scoped_lock lock(connection_mutex_);
     ros::SubscriberStatusCallback connect_cb
-      = boost::bind(&NodeletLazy::connectionCallback, this, _1);
-    ros::SubscriberStatusCallback disconnect_cb
-      = boost::bind(&NodeletLazy::connectionCallback, this, _1);
+      = boost::bind(&NodeletLazy::connectionCallback, this);
```

Comment:
Note that I remove the `_1` placeholder here, which is important after the change in `connectionCallback`.

The change to use `connect_cb` for both disconnection and connection is optional, but I think it's cleaner, since this was already declaring `disconnect_cb` the same way as `connect_cb`.